js/routes.js

```diff
@@ -0,0 +1,6 @@
+page('/about', aboutController.index);
```

Comment:
You probably want to set up a route for just '/', since right now the behavior on the base page is odd.